src/utils.tsx

```diff
@@ -61,3 +65,13 @@ export const getPathFromQueryString = (history: History) => {
 
   return typeof path === 'string' && path.length ? path : null;
 };
+
+export const createCodeLineAnchorGetter = (
+  compareInfo: CompareInfo | null | void,
```

Comment:
Sure, I can do that. I guess I'm just inclined to not use an object when there's only a single argument, but the fact that it can be `undefined` is valid.